mmtrack/datasets/transforms/formatting.py

```diff
@@ -403,7 +403,7 @@ class PackReIDInputs(BaseTransform):
             ``metainfo`` of the packed ``data_sample``.
     """
     default_meta_keys = ('img_path', 'ori_shape', 'img_shape', 'scale',
-                         'scale_factor', 'flip', 'flip_direction')
+                         'scale_factor')
```

Comment:
I don't get the purpose of the pr.
It seems nothing changed

Reply:
After these changes, the keys `'flip', 'flip_direction'` are only used in training pipeline, not for testing.
Otherwise, there will be a `Key error` of `flip` while testing the ReID model.

Please wait for the results of regression benchmark before merging this pr.

Reply:
done.
the result is:
`reid-metric/mAP: 0.6220  reid-metric/R1: 0.9980  reid-metric/R5: 0.9990  reid-metric/R10: 0.9990`

Reply:
Is it ok to merge this pr?